### PR TITLE
added base

### LINKED ISSUE #444: Add base chain support
Add Base chain wallet support
Add THORChain Dex support

### PR DESCRIPTION
**Pull request by Thorian1te**

### CODE REVIEW

File: src/main/api/ledger/evm/address.ts [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+  transport: Transport
+  walletAccount: number
+  walletIndex: number
+  evmHDMode: EvmHDMode
+  network: Network
+}): Promise<E.Either<LedgerError, WalletAddress>> => {
+  let clientParams
+
+  switch (chain) {
+    case 'ETH':
```
Comment (cinnamoroll6130) [inline src/main/api/ledger/evm/address.ts]: What about `ETHChain`, `ARBChain`, etc instead of string?

File: src/renderer/App.tsx [hunk 2]
```diff
@@ -44,43 +45,46 @@ export const App: React.FC = (): JSX.Element => {
...
-                                      </CosmosProvider>
-                                    </DashProvider>
-                                  </XrdProvider>
-                                </SolProvider>
-                              </KujiProvider>
-                            </DogeProvider>
-                          </ArbProvider>
-                        </BscProvider>
+                        <BaseProvider>
+                          {' '}
```
Comment (cinnamoroll6130) [inline src/renderer/App.tsx]: Need to remove?

File: src/renderer/services/evm/const.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { ethers } from 'ethers'
+
+export const ETH_OUT_TX_GAS_LIMIT = ethers.BigNumber.from('35609')
+export const ERC20_OUT_TX_GAS_LIMIT = ethers.BigNumber.from('49610')
+export const DEPOSIT_EXPIRATION_OFFSET = 15 * 60 // 15min in seconds
+export const zeroAddress = '0x0000000000000000000000000000000000000000'
```
Comment (cinnamoroll6130) [inline src/renderer/services/evm/const.ts]: it'd be great to start with capital letter - `ZeroAddress `or `EVMZeroAddress`.

File: src/main/api/ledger/evm/address.ts (hunk 1 above)
Comment (Thorian1te) [reply src/main/api/ledger/evm/address.ts]: yep

File: src/renderer/App.tsx (hunk 2 above)
Comment (Thorian1te) [reply src/renderer/App.tsx]: yep

File: src/renderer/services/evm/const.ts (hunk 3 above)
Comment (Thorian1te) [reply src/renderer/services/evm/const.ts]: sure

**Review by cinnamoroll6130 — DISMISSED** [review]
Left some comments

**Review by cinnamoroll6130 — DISMISSED** [review]
lgtm

**Review by cinnamoroll6130 — APPROVED** [review]

**Pull request merged**